common/src/jni/main/include/macros.h

```diff
@@ -17,14 +17,13 @@
 #ifndef CONSCRYPT_SRC_MAIN_NATIVE_MACROS_H_
 #define CONSCRYPT_SRC_MAIN_NATIVE_MACROS_H_
 
-#ifdef JNI_JARJAR_PREFIX
-#define TO_STRING(x) #x
-#define CONSCRYPT_SYMBOL_PREFIX TO_STRING(JNI_JARJAR_PREFIX)
-#else
-#define CONSCRYPT_SYMBOL_PREFIX ""
+#define TO_STRING1(x) #x
+#define TO_STRING(x) TO_STRING1(x)
+#ifndef JNI_JARJAR_PREFIX
 #ifndef CONSCRYPT_NOT_UNBUNDLED
 #define CONSCRYPT_UNBUNDLED
 #endif
+#define JNI_JARJAR_PREFIX
```

Comment:
IIRC, "" doesn't work either ... I forget why exactly.  I had tried a few things here ... it's a bit tricky.